R/data_to_long.R

```diff
@@ -152,23 +153,21 @@ data_to_long <- function(data,
       exclude = NULL,
       ignore_case = ignore_case,
       regex = regex,
-      verbose = FALSE
+      verbose = verbose
+    )
+  } else if (!missing(select) || !is.null(select)) {
+    cols <- .select_nse(
+      select,
+      data,
+      exclude = NULL,
+      ignore_case = ignore_case,
+      regex = regex,
+      verbose = verbose
```

Comment:
Should we add `ifnotfound = "error"` (added not that long ago) here? Pivoting on columns that don't exist will lead to nonsensical results, so erroring early would prevent footguns IMO

Reply:
Agree. But then we don't need `verbose`.